coding/browser-support.md

```diff
@@ -0,0 +1,36 @@
+# Browser support
+
+Our baseline browser support for our websites. There are two levels of support. 
+
+## Full support
+
+Any browser with full support needs to be pixel perfect and all functionality 
+should work.
```

Comment:
Maybe it is just me but could you put paragraph on one line? The IDE should manage the returns.

Reply:
I don't think we have to have such strict styling rules, especially for writing Markdown. It just slows work down unnecessarily.

I also don't agree that this would be a good standard to have. It's part of the markdown standard that you can wrap paragraphs over multiple lines, and I think it should be fine to do so. In Python we enforce wrapping at 80 chars, so it's not unprecedented. I think we should just learn to read text that might have wrapped.

Reply:
We recently discussed this: https://github.com/ubuntudesign/meeting-notes/issues/182 (last section)

There was a general preference not to hard wrap text. It can be annoying when you have to shuffle all the text around, along with messier diffs it creates.

Reply:
Well remembered @WillMoggridge. I didn't recall that at all. I don't mind someone telling @anthonydillon to change how he writes things, but I do generally stand by what I said.

I do think that one of the secrets to our success as a team, compared to some other teams, is that we try to stay nice and light on bureaucracy. Hopefully it means we're spending more time discussing the actual work we do, rather than how we do it.

To me it seems more efficient to teach our team members to be flexible to others' different ways of working except where it really matters, rather than trying to teach every member of our team (current & future) to do everything in exactly the same way. But unfortunately we could just end up arguing until we cows come home about where we put that "really matters" line :(.